app/src/main/java/com/eveningoutpost/dexdrip/Models/BloodTest.java

```diff
@@ -124,6 +124,10 @@ private BloodTestMessage toMessageNative() {
     private static final long CLOSEST_READING_MS = 30000; // 30 seconds
```

Comment:
Can you explain what the changes in BloodTest do? It looks like the treatment de-duplication handling that I introduced for treatments but applied to bloodtests. This probably makes sense, but is this only applied to watch synced items?